### ISSUE TITLE
Microformat

### ISSUE THREAD

**Issue by colbyfayock** [colbyfayock/next-wordpress-starter#44]
We should add logical microformats throughout the site

http://microformats.org/
https://microformats.io/
https://developer.mozilla.org/en-US/docs/Web/HTML/microformats

**Comment by arifsetyo21** [comment]
which part to add microformat?

**Comment by colbyfayock** [comment]
hey @arifsetyo21 i'm hoping to add as much coverage as makes sense, so that includes author information, post information, website information. it would be great if as part of this issue, the person who takes it tries to figure out what all make sense

**Comment by arifsetyo21** [comment]
Okay, can i do this issue?

**Comment by colbyfayock** [comment]
sure thing :) let me know if you have any issues!

**Comment by doingandlearning** [comment]
I don't know if this belongs in this issue but it would be good to have everything required for webmentions leveraging of microformats in here.

**Comment by doingandlearning** [comment]
(I'll find a link!)

**Comment by colbyfayock** [comment]
yes! that was one of the things i had in mind when thinking about it. it would be cool as that information is valuable for services that can provide a rich experience and if im not mistaken even search engines use it

**Comment by colbyfayock** [comment]
hey @arifsetyo21 checking in - are you having any luck with this?

**Comment by Azanul** [comment]
@colbyfayock I'll take this up if no one else is working on it. Also, I think this issue should be split in two as there could be too many changes at once.

**Comment by colbyfayock** [comment]
@Azanul sounds good to me! would love to hear your thoughts then before moving forward including the work that needs to be done and what you see as the 2 issues

**Comment by Azanul** [comment]
@colbyfayock I meant 2 PRs as there can be many changes.

**Comment by colbyfayock** [comment]
Got it, I would still like 2 issues to have concise details around the problems each are solving so that we're not creating 2 PRs for the sake of creating 2 PRs

**Comment by colbyfayock** [comment]
from what i understand i think thats part of it. i was also under the impression there would be more attributes than classes, such as "roles" and such, but i haven't looked too deeply into it

**Comment by Azanul** [comment]
You mean adding `rel` attribute?

**Comment by Azanul** [comment]
Let's divide it in either of these two ways:
1. per component
2. all components per required task (i.e., `rel` attribute and class name)

**Comment by colbyfayock** [comment]
i don't think it makes sense to create a new PR for every component, but if you'd like to break up the work another way, such as one for pages and one for components, I'm open to that

**Comment by Azanul** [comment]
Not one PR for each component but one for half of them and another for other half

**Comment by colbyfayock** [comment]
let's do pages vs components as that sounds a little bit like a more logical difference

/src/pages
vs 
/src/components

that sound good?

**Comment by Azanul** [comment]
By vs you mean one PR for pages and one for components right?

**Comment by Azanul** [comment]
Let me take up components then

**Comment by colbyfayock** [comment]
yup, sounds great
